.github/workflows/labeler.yaml

```diff
@@ -0,0 +1,52 @@
+name: Auto Label Issues and PRs
+
+on:
+  issues:
+    types: [opened, edited, reopened, transferred]
+  issue_comment:
+    types: [created, edited]
+  pull_request_target:
+    types: [opened, edited, reopened]
+
+jobs:
+  label:
+    permissions:
+      contents: read
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        
+      - name: Get changed files
+        if: ${{ github.event.pull_request }}
+        id: changed-files
+        uses: tj-actions/changed-files@ed68ef82c095e0d48ec87eccea555d944a631a4c
+        with:
+          separator: ","
+          
+      - name: Run labeler action for PR
+        if: ${{ github.event.pull_request }}
+        uses: cncf/automation/.github/actions/labeler-action@main
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          labels: ./.github/labels.yaml
```

Comment:
This seems to be expected to be a URL: https://github.com/cncf/automation/blob/main/utilities/labeler/main.go#L101

```
 2025/08/06 21:36:44 failed to load labels.yaml: failed to fetch labels.yaml from URL: Get "./.github/labels.yaml": unsupported protocol scheme ""
```

Maybe this needs to be `file://./.github/labels.yaml`?

Would you like a PR?